index.html

```diff
@@ -1,76 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-   <meta charset="UTF-8">
-    <title></title>
-    <link rel="stylesheet" href="./css/style.css">
-    <link rel="stylesheet" href="./css/reset.css">
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
-
```

Comment:
Please do not leave an unnecessary comment in your codebase as this may just confuse and can make things unclear.